content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -8,15 +8,14 @@ content_type: concept
 weight: 60
 aliases:
 - controller-metrics.md
+- monitoring.md
```

Comment:
Use two Netlify redirects here, rather than Hugo aliases.

Reply:
👍

Reply:
See the `static/_redirects` file.

Reply:
Added, not sure if it's correct.

Reply:
How about removing the alias, `controller-metrics.md`? Just add another line to the _redirect file.
I looked quickly; the system-metrics entry looks OK.

Reply:
Fixed